app/screens/home/RouteList.tsx

```diff
@@ -172,12 +183,12 @@ const RouteList = ({
       .finally(() => setIsStakingLoading((obj) => updateObject(obj, { [route.id]: false })));
   };
   const deleteCryptoRoute = (route: CryptoRoute) => {
-    setIsCryptoLoading((obj) => updateObject(obj, { [route.id]: true }))
+    setIsCryptoLoading((obj) => updateObject(obj, { [route.id]: true }));
     return putCryptoRoute(updateObject(route, { active: false }))
       .then(() => (route.active = false))
       .catch(() => NotificationService.error(t("feedback.delete_failed")))
       .finally(() => setIsCryptoLoading((obj) => updateObject(obj, { [route.id]: false })));
-  }
+  };
```

Comment:
Maybe it is just me and coming from other languages, but using `;` feels weird. Most languages neglected this usage as it is not needed. I personally heavy prefer to not use it either

Reply:
That's default Prettier setting. Feel free to define custom rules and impose them on us (as long as it is auto formatted) ... I'm open for everything 😁

Reply:
hahhaha oh shit, you are right. i mean if i haven't to write them. I don't really care. Lets see what the future brings in